Weather/WeatherForecast/harvest/portugal_weather_forecast_harvest.py

```diff
@@ -70,14 +70,8 @@ def decode_wind_direction(direction):
 
 
 def get_weather_forecasted():
-    data = {}
-
-    for locality in iptma_codes:
-        code = iptma_codes[locality]
-
-        data[code] = get_weather_forecasted_pt(locality)
-
-    return data
+    return {iptma_codes[locality]: get_weather_forecasted_pt(locality)
```

Comment:
even though I appreciate the effort to make it more Pythonic, I think the original code is more evident for someone who is not a great Python expert